src/caliper/CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ set_target_properties(caliper PROPERTIES SOVERSION ${CALIPER_MAJOR_VERSION})
 set_target_properties(caliper PROPERTIES VERSION ${CALIPER_VERSION})
 
 target_link_libraries(caliper PUBLIC caliper-common)
+target_link_libraries(caliper PUBLIC numa)
```

Comment:
TODO: fix

Reply:
@daboehme : excuse the review of my own code, that was just the most natural way to make possibly multiple comments